Kajabi/sage-lib
# Responsiveness Column in Status

## Description

Adds a responsiveness column to the Sage docs Status page. Also alphabetizes components within the `component_helper.rb` file.

## Screenshots

<img width="1293" alt="Screen Shot 2022-06-03 at 12 51 10 PM" src="https://user-images.githubusercontent.com/14791307/171920498-3d13a0ef-f5f1-4980-8a80-083c73d5d064.png">


## Testing in `sage-lib`

- Navigate to the [status page](http://localhost:4000/pages/status)
- Check that the responsiveness column is present with all components listed as "todo."

## Testing in `kajabi-products`

(**LOW**) Adds a responsiveness column to the Sage docs Status page. Also alphabetize components within the `component_helper.rb` file.
